services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/LegAnnotation.java

```diff
@@ -219,6 +223,28 @@ public abstract static class Builder extends DirectionsJsonObject.Builder<Builde
     @NonNull
     public abstract Builder trafficTendency(@Nullable List<Integer> trafficTendency);
 
+    /**
+     * Use this method to add parameters which are not present in the model yet but are supported
+     * on the Directions API side in the response.
+     * Use it for experimental parameters.
+     *
+     * @param unrecognizedProperties parameters to add to request
+     */
+    @NonNull
+    public Builder unrecognizedProperties(
```

Comment:
Why not make the abstract `unrecognized` method public in `DirectionsJsonObject.Builder`? We definitely see a use-case for this method being public. Or add `unrecognizedProperties` as public function to the abstract builder if we don't want to expose the `SerializableJsonElement`.

cc @VysotskiVadim

Reply:
1) It wasn't public so I thought there was a reason for it. So we should only make it public when needed, a white list, not black list;
2) The `unrecognized()` method accepts `Map<String, SerializableJsonElement>` which is not very convenient for the end user, so yes, I'd prefer adding `unrecognizedProperties` with `Map<String, JsonElement>` instead.

So is there a reason why it's not public?